modules/core/src/main/java/org/apache/ignite/internal/binary/BinarySchema.java

```diff
@@ -25,10 +25,7 @@
 import java.io.IOException;
 import java.io.ObjectInput;
 import java.io.ObjectOutput;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
Ignite code convention doesn't allow asterisk in imports.

https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-PackageImporting